doc/api.rst

```diff
@@ -0,0 +1,43 @@
+.. _api_ref:
+
+###
+API
+###
+
+:py:mod:`pywhy_stats`:
+
+.. automodule:: pywhy_stats
+   :no-members:
+   :no-inherited-members:
+
+This is the application programming interface (API) reference
+for classes (``CamelCase`` names) and functions
+(``underscore_case`` names) of pywhy-stats, grouped thematically by analysis
+stage.
+
+********************
+Independence Testing
+********************
+Dodiscover experimentally provides an interface for conditional independence
+testing and conditional discrepancy testing (also known as k-sample conditional
+independence testing).
+
+Conditional Independence Testing
+================================
+
+Testing for conditional independence among variables is a core part
+of many data analysis procedures.
+
+Conditional k-sample testing
+============================
+
+Testing for conditional discrepancy among variables is a core part
+of many causal inference procedures, such as constraint-based structure
+learning.
+
+Utilities
+=========
+
+Testing for conditional discrepancy among variables is a core part
+of many causal inference procedures, such as constraint-based structure
+learning.
```

Comment:
Repetition of the above. What are `Utilities` here?

Reply:
Removing for now.